.github/workflows/storybook-deploy.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: ['12', '14', '16']
```

Comment:
one version only, since again this is just to use a "tool" node version to publish the storybook

Reply:
changed and squashed